### PR TITLE
Best of 3 tournaments

### PR DESCRIPTION
**Pull request by gesundkrank**

### CODE REVIEW

File: src/main/java/eu/m6r/kicker/Store.java [hunk 1]
```diff
@@ -93,17 +93,18 @@ private Team getTeam(final Player player1, final Player player2) {
...
-    }
-
-    public void addMatch(final int tournamentId) {
+    public void addMatch(final int tournamentId) throws InvalidTournamentStateException {
         final Transaction tx = session.beginTransaction();
 
         final Tournament tournament = session.get(Tournament.class, tournamentId);
 
-        addMatch(tournament);
+        if (tournament.bestOfN <= tournament.matches.size()) {
```
Comment (hvoecking) [inline src/main/java/eu/m6r/kicker/Store.java]: Brauchen wir die `bestOfN` variable überhaupt noch? Ist doch eigentlich klar, dass es immer best-of-3 ist oder?

File: src/main/java/eu/m6r/kicker/Controller.java [hunk 2]
```diff
@@ -112,9 +112,12 @@ public String addPlayer(Player player)
         players.add(player);
 
         if (players.size() == 4) {
-            final String players = getPlayersString();
             startTournament();
-            return String.format("%s a new game started!", players);
+
+            Tournament tournament = getRunningTournament();
+            return String.format("A new game started:%n <@%s> <@%s> vs. <@%s> <@%s>",
```
Comment (hvoecking) [inline src/main/java/eu/m6r/kicker/Controller.java:118]: Can you also add the team colors here?

File: src/main/java/eu/m6r/kicker/Store.java (hunk 1 above)
Comment (gesundkrank) [reply src/main/java/eu/m6r/kicker/Store.java]: No. I want to keep it configurable.

File: src/main/java/eu/m6r/kicker/Controller.java (hunk 2 above)
Comment (gesundkrank) [reply src/main/java/eu/m6r/kicker/Controller.java:118]: I think the left team is the one that starts in left side. :)

**Comment by gesundkrank** [comment]
Added a fixup that improves the best-of-n check.

**Pull request merged**